docs/guides/joint_fabric_guide.md

```diff
@@ -250,4 +253,16 @@ jf-admin-app has been installed:
 A `Subjects` field equal to `18446744065119551489` (`FFFFFFFDFFFF0001` in hex)
 should be found.
 
+-   Open Joint Commissioning Window on JF Admin App of Ecosystem B
+
+```
+>>> pairing open-joint-commissioning-window 1 400 1000 1261
+```
+
+Check for the following logs on the jf-admin-app side:
+
+```
+>>> [DIS] Advertise commission parameter vendorID=65521 productID=32769 discriminator=1261/04 cm=2 cp=0 jf=14
```

Comment:
jf-admin-app from Ecosystem B has 0xFFF2 Vendor ID (see the instructions where it has been commissioned using --commissioner-vendor-id 0xFFF2). Why does it advertise a vendorID of 65521 (0XFFF1)?

Reply:
--commissioner-vendor-id only sets the vendor ID of the commissioner, not the device. To set the vendor ID on the device side, you need to add --vendor-id to the command that launches the jfb-admin-app.

Reply:
I understand, makes sense. In this case please use --vendor-id 0xFFF1 as parameter for jf-admin-app on Ecosystem 1 and --vendor-id 0xFFF2 as parameter for jf-admin-app on Ecosystem 2. 

The --commissioner-vendor-id and  --vendor-id must be identical for the jf-control-app and jf-admin-app of the same Ecosystem.